test_app/models.py

```diff
@@ -54,6 +54,12 @@ class Meta(AbstractDABUser.Meta):
     def summary_fields(self):
         return user_summary_fields(self)
 
+    def get_authenticator_ids(self) -> list[int]:
+        return list(self.authenticator_users.values_list('provider__id', flat=True))
+
+    def get_authenticator_uids(self) -> list[str]:
+        return list(self.authenticator_users.values_list('uid', flat=True).distinct())
+
```

Comment:
@relrod we should think if there is a way we can auto-add these methods to the default User object if dab_authentication is enabled.